determined-ai/determined
# chore: add lint for Prettier and Ignore format code commit

## Description
- Related to #5015 
- This is Step3 of https://github.com/determined-ai/determined/pull/5015#issuecomment-1248376734
- [DET-8097]



## Test Plan
- Check if code works as expected
- Check if all test cases pass
- Check if the right commit is ignored



## Commentary (optional)




## Checklist
See [Release Note](https://github.com/determined-ai/determined/blob/master/docs/release-notes/README.md) for details.





[DET-1234]: https://determinedai.atlassian.net/browse/DET-1234?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

[DET-8097]: https://determinedai.atlassian.net/browse/DET-8097?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
